src/traits.rs

```diff
@@ -1,18 +1,18 @@
 // Take a look at the license at the top of the repository in the LICENSE file.
 
 use crate::{
-    common::{Gid, Uid},
+    common::{Gid, MacAddr, Uid},
     sys::{Component, Cpu, Disk, Networks, Process},
 };
 use crate::{
     CpuRefreshKind, DiskType, DiskUsage, LoadAvg, NetworksIter, Pid, ProcessRefreshKind,
     ProcessStatus, RefreshKind, Signal, User,
 };
 
-use std::collections::HashMap;
 use std::ffi::OsStr;
 use std::fmt::Debug;
 use std::path::Path;
+use std::{collections::HashMap, net::Ipv4Addr};
```

Comment:
Please add instead `use std::net::Ipv4Addr;` rather than joining two imports that don't belong in the same module.